src/renderer/components/ft-list-channel/ft-list-channel.js

```diff
@@ -53,11 +53,8 @@ export default defineComponent({
 
       this.channelName = this.data.name
       this.id = this.data.id
-      if (this.hideChannelSubscriptions || this.data.subscribers == null) {
-        this.subscriberCount = null
-      } else {
-        this.subscriberCount = this.data.subscribers.replace(/ subscriber(s)?/, '')
-      }
+      this.subscriberCount = this.data.subscribers !== null ? this.data.subscribers.replace(/ subscriber(s)?/, '') : ''
```

Comment:
Original code is `== null`, the opposite should be `!= null`
```suggestion
      this.subscriberCount = this.data.subscribers != null ? this.data.subscribers.replace(/ subscriber(s)?/, '') : ''
```

Reply:
Thanks, that could have caused a nasty bug. It should be fixed in the latest commit